staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -108,6 +108,11 @@ func (d *validatingDispatcher) callHook(ctx context.Context, h *v1beta1.Webhook,
 		}
 	}
 
+	// Currently dispatcher only supports `v1beta1` AdmissionReview
+	if !util.HasAdmissionReviewVersion("v1beta1", h) {
```

Comment:
prefer v1beta1.SchemeGroupVersion.Version, and add a TODO to make the dispatcher capable of sending/receiving multiple versions